slackbridge/bots.py

```diff
@@ -89,6 +89,18 @@ def check_slack_rtm(self):
             elif message['presence'] == 'active':
                 user_bot.back()
             return
+        elif (message['type'] == 'member_joined_channel' and
+                message['user'] in self.users):
+            self._rtm_channel_helper(message)
+            return
+        elif (message['type'] == 'member_left_channel' and
+                message['user'] in self.users):
+            self._rtm_channel_helper(message, False)
+            return
+        elif (message['type'] == 'team_join'):
+            self.slack_users.append(message['user'])
```

Comment:
I think making this `IRCBot.slack_users.append(message['user'])` would make it more clear that it's modifying state that other bots can access